### DISCUSSION TITLE
Can you re-support old version WLK

### DISCUSSION THREAD

**Discussion by DiamoJun** [simulationcraft/simc#6931] [Q&A]
There are a lot of changes in this WLK.
Can your guys support and release plugins

**Comment by nyterage** [comment]
Simulationcraft is an open source project. Anyone who works on it is a volunteer who works on whatever they wish to help with. Currently, no one on the team has any plans to work on WotLK Classic for Simulationcraft, although its not out of the question, its unlikely to ever happen as the classic community has built their own tools for simulations.

**Reply by DiamoJun** [reply]
thank you for your reply
